sdccc/src/main/java/com/draeger/medical/sdccc/tests/glue/direct/DirectSubscriptionHandlingTest.java

```diff
@@ -429,6 +492,7 @@ private Duration getSubscriptionStatus(final ReportTestData report) {
                         "encountered exception while querying the subscription status of report "
                                 + report.getReportName(),
                         e);
+                // TODO: be more strict. Do not tolerate Errors any more.
```

Comment:
what's up with this TODO? when is it supposed to be done?

Reply:
This is obsolete. When I noticed that SDC.ri answered GetStatus with a 404 status code, I realized this was broken and this was my first idea how to handle this situation. However, the test case expects a SubscriptionEnd-Message now and after a subscription has officially been cancelled using such a message,  it is ok to answer a GetStatus request with 404. Hence, I will remove this TODO.

Reply:
Done.